README.md

```diff
@@ -68,6 +68,7 @@ npm install --save @dreamworld/dw-list-item
 | Name  | Description |
 | ----  | ----------- |
 | --dw-list-item-hover-background | Background color of the hovered item  |
```

Comment:
Custom CSS Properties declared here isn't used in the source code. Let's make the documentation in the sync with the current implementation.